tools/gulp/gulpfile.ts

```diff
@@ -1,16 +1,21 @@
+import {createPackageBuildTasks} from './util/package-tasks';
+
+/** Create gulp tasks to build the different packages in the project. */
+createPackageBuildTasks('cdk');
+createPackageBuildTasks('material', ['cdk']);
+createPackageBuildTasks('material-examples', ['material']);
```

Comment:
I totally see what you mean. My only concern is that it is very confusing to have a mismatch of package-name / task-name / source directory name.